app/views/admin_applications/show.html.erb

```diff
@@ -0,0 +1,31 @@
+<h1> Application <%= @application.id %></h1>
+
+<%= render "layouts/app_info" %>
+
+<h2> Pets Applied For: </h2>
+
+<% @application_pets.each do |application_pet| %>
+  <% if application_pet.pet_status != "Approved" && application_pet.pet_status != "Rejected" %> 
+    <h3><%= application_pet.pet.name %></h3>
+    <%= form_with url: "/admin/application_pets/#{application_pet.id}", method: :patch, data: {turbo:false} do |form| %>
```

Comment:
Talked about this a little in your `routes.rb` file, and while this route is sort of restful, the path is not quite conventional.

```ruby
<%= form_with url: "/admin/applications/#{application_pet.application_id}/pets/#{application_pet.pet_id}", method: :patch, data: {turbo:false} do |form| %>
```